docs/source/examples/spot-jobs.rst

```diff
@@ -306,3 +306,72 @@ The :code:`resources` field has the same spec as a normal SkyPilot job; see `her
   These settings will not take effect if you have an existing controller (either
   stopped or live).  For them to take effect, tear down the existing controller
   first, which requires all in-progress spot jobs to finish or be canceled.
+
+
+Spot Pipeline
+-------------------------
+
+Spot Pipeline is a feature that allows you to submit a spot job that contains a sequence of spot tasks running one after another.
+This is useful for running a sequence of jobs that depend on each other, e.g., training a model and then running inference on it.
+This allows the multiple tasks to have different resource requirements to fully utilize the resources and save cost, while keep the burden of managing the tasks off the user. 
```

Comment:
```suggestion
This allows the multiple tasks to have different resource requirements to fully utilize the resources and save cost, while keeping the burden of managing the tasks off the user. 
```

Perhaps add a note that all tasks in a pipeline are run on spot instances and we currently don't support on-demand instances for Spot Pipeline.